lib/mayaUsd/utils/util.h

```diff
@@ -656,6 +656,29 @@ MString GetCurrentMayaWorkspacePath();
 MAYAUSD_CORE_PUBLIC
 MString GetCurrentSceneFilePath();
 
+/**
+ * Returns all the sublayers recursively for a given layer
+ *
+ * @param layer   The layer to start from and get the sublayers
```

Comment:
I noticed you are mixing the kinds of doxygen tags you are using. Here you use @ and above you use \

Reply:
Honestly I try to follow the style of the file I'm in